Rakefile

```diff
@@ -6,5 +6,16 @@ task :environment do
   # require './config/environment'
 end
 
+scss = "public/scss/main.scss"
+css = "public/css/main.css"
+
+task :watch do
+  sh "bundle exec sass --watch #{scss}:#{css}"
+end
+
+task :clean do
+  sh "rm -f #{css}"
```

Comment:
If you're interested, Rake has some neat built-ins for this, such as `rm_rf css`. This removes the need to shell out.